thundra/wrappers/aws_lambda/lambda_executor.py

```diff
@@ -1,18 +1,18 @@
 import math
-import time
 import uuid
 
 from thundra import utils, constants
 from thundra.config import config_names
 from thundra.config.config_provider import ConfigProvider
 from thundra.plugins.invocation import invocation_trace_support
 from thundra.plugins.log.thundra_logger import debug_logger
+from thundra.wrappers import wrapper_utils
 from thundra.wrappers.aws_lambda import lambda_event_utils
 
 
 def start_trace(plugin_context, execution_context, tracer):
-    set_start_time(execution_context)
     context = execution_context.platform_data['originalContext']
+    execution_context.trace_id = str(uuid.uuid4())
```

Comment:
Why we set trace id here? Isn't it set somewhere else by considering span context propagation?